antelope_contracts/contracts/erc20/include/erc20/erc20.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <eosio/asset.hpp>
+#include <eosio/eosio.hpp>
+#include <eosio/singleton.hpp>
+#include <intx/intx.hpp>
+#include <erc20/types.hpp>
+
+using namespace eosio;
+using namespace intx;
+
+namespace erc20 {
+
+class [[eosio::contract]] erc20 : public contract {
+   public:
+    using contract::contract;
+
+    [[eosio::on_notify("*::transfer")]] void transfer(eosio::name from, eosio::name to, eosio::asset quantity, std::string memo);
+
+    // evm runtime will call this to notify erc20 about the message from 'from' with 'data'.
+    [[eosio::action]] void onbridgemsg(name receiver, const bytes& sender, const time_point& timestamp, const bytes& value, const bytes& data);
```

Comment:
This is still using the old interface of the `onbridgemsg` action that didn't take an `std::variant`.

@taokayan: I am assuming you have already fixed this in your branch? If so, there is no need to change this within this PR.

Reply:
i will merge without this fix so kayan can merge in his fixes.

Reply:
yes it is using std::variant in my local branch